app/models/identity.rb

```diff
@@ -4,12 +4,14 @@ class Identity < ApplicationRecord
   belongs_to :user
   validates :service_provider, presence: true
 
+  delegate :metadata, to: :sp, prefix: true
```

Comment:
Oh, cool.  I didn't know this existed, makes total sense.

Reply:
Honestly, I don't like it (because the resulting method is called `sp_metadata` and that's nowhere clear on this line..... but Rubocop enforced it 🤖